app/models/assignment.rb

```diff
@@ -22,6 +22,7 @@ class Assignment < ActiveRecord::Base
   belongs_to :course
   belongs_to :article
   belongs_to :wiki
+  has_many :feedbacks
```

Comment:
If these are *only* linked to an assignment, they should be destroyed if the assignment is destroyed.